src/features/import/components/Configure/Preview/index.tsx

```diff
@@ -0,0 +1,174 @@
+import OrgPreview from './OrgPreview';
+import PreviewGrid from './PreviewGrid';
+import TagsPreview from './TagsPreview';
+import { useNumericRouteParams } from 'core/hooks';
+import usePersonPreview from 'features/import/hooks/usePersonPreview';
+import useSheets from 'features/import/hooks/useSheets';
+import { ColumnKind, Sheet } from 'features/import/utils/types';
+
+const Preview = () => {
+  const theme = useTheme();
+  const { sheets, selectedSheetIndex, firstRowIsHeaders } = useSheets();
+  const [personIndex, setPersonIndex] = useState(0);
+  const currentSheet: Sheet = sheets[selectedSheetIndex];
+  const { orgId } = useNumericRouteParams();
+  const { fields, org, tags } = usePersonPreview(
+    currentSheet,
+    firstRowIsHeaders ? personIndex + 1 : personIndex,
+    orgId
+  );
+
+  const emptyPreview = currentSheet.columns.every(
+    (item) => item.selected === false
+  );
+
+  const tagColumnSelected = currentSheet.columns.some(
+    (column) => column.kind === ColumnKind.TAG && column.selected
+  );
+
+  useEffect(() => {
+    setPersonIndex(0);
+  }, [selectedSheetIndex]);
+
+  return (
+    <Box p={2}>
+      <Box alignItems="center" display="flex" sx={{ mb: 1.5 }}>
+        <Typography sx={{ mr: 2 }} variant="h5">
+          <Msg id={messageIds.configuration.preview.title} />
+        </Typography>
+        <Button
+          disabled={personIndex === 0}
+          onClick={() =>
+            setPersonIndex((prev) => (personIndex !== 0 ? prev - 1 : prev))
+          }
+          startIcon={<ArrowBackIos />}
+        >
+          <Msg id={messageIds.configuration.preview.previous} />
+        </Button>
+        <Button
+          disabled={
+            personIndex ===
+            currentSheet.rows.length - (firstRowIsHeaders ? 2 : 1)
+          }
+          endIcon={<ArrowForwardIos />}
+          onClick={() =>
+            setPersonIndex((prev) =>
+              personIndex < currentSheet.rows.length - 1 ? prev + 1 : prev
+            )
+          }
+        >
+          <Msg id={messageIds.configuration.preview.next} />
+        </Button>
+      </Box>
+      <Box
+        sx={{
+          alignItems: 'center',
+          border: '1px solid lightgrey',
+          borderRadius: '5px',
+          display: 'flex',
+          height: '98px',
+          overflowX: 'auto',
+          overflowY: 'hidden',
+        }}
+      >
+        <Box
+          display="flex"
+          justifyContent="space-between"
+          sx={{ width: '100%' }}
+        >
+          {emptyPreview &&
+            Array(currentSheet.columns.length)
+              .fill(2)
+              .map((marginSize, index) => {
+                return (
+                  <Box
+                    key={`empty-preview-${index}`}
+                    flexGrow={1}
+                    sx={{
+                      backgroundColor: theme.palette.transparentGrey.light,
+                      height: '14px',
+                      m: marginSize,
```

Comment:
Oh, I see now you use the `2` from the array here, but why not just hard-code this as `2` instead of hard-coding it on line 88? Hard-coding it here would be more readable.